From 9c17deb0526b90b834bd421db872737c96f7227c Mon Sep 17 00:00:00 2001
From: Michael Terry <mike@mterry.name>
Date: Sun, 24 Apr 2022 10:59:56 -0400
Subject: [PATCH] build: add gtk wayland and x11 dep objects to deja-dup
 executable

This fixes our build with vala 0.56.1+, which split those vapi
files out from the main gtk4 vapi file.

Fixes: https://gitlab.gnome.org/World/deja-dup/-/issues/294
---
 app/meson.build | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/app/meson.build b/app/meson.build
index d2cfb394..6e732bf7 100644
--- a/app/meson.build
+++ b/app/meson.build
@@ -50,6 +50,8 @@ deja_dup = executable('deja-dup',
         adwaita_dep,
         gio_unix_dep,
         gtk_dep,
+        gtk_wayland_dep,
+        gtk_x11_dep,
         packagekit_dep,
         secret_dep,
     ],
-- 
GitLab

